src/TLuaInterpreter.cpp

```diff
@@ -3657,26 +3657,61 @@ int TLuaInterpreter::getMudletInfo(lua_State* L)
 {
     Host& host = getHostFromLua(L);
 
-    QStringList knownEncodings{};
-    {
-        auto adjustEncoding = [](auto encodingName) {
-            auto originalEncoding = encodingName;
-            if (encodingName.startsWith("M_")) {
-                encodingName.remove(0, 2);
-            }
-
-            return (originalEncoding == encodingName) ? originalEncoding : QStringLiteral("%1 (%2)").arg(encodingName, originalEncoding);
-        };
-        for (const auto& encoding : host.mTelnet.getEncodingsList()) {
-            knownEncodings.append(adjustEncoding(encoding));
+    QByteArrayList knownEncodings{"\"ASCII\""};
+    auto adjustEncoding = [](auto encodingName) {
+        auto originalEncoding = encodingName;
+        if (encodingName.startsWith("M_")) {
+            encodingName.remove(0, 2);
         }
-        knownEncodings.sort(Qt::CaseInsensitive);
-    }
 
-    host.postMessage(tr("[ INFO ]  - Available encodings:"));
-    host.postMessage(QStringLiteral("  %1").arg(knownEncodings.join(QStringLiteral(", "))));
+        return (originalEncoding == encodingName) ? "\"" + originalEncoding + "\""
+                                                  : ("\"" + encodingName + "\" (\"" + originalEncoding + "\")");
+    };
+    for (const auto& encoding : host.mTelnet.getEncodingsList()) {
+        knownEncodings.append(adjustEncoding(encoding));
+    }
+
+    QString encodingNames{QLatin1String(knownEncodings.join(", "))};
```

Comment:
I don't think it's this PR's scope to rework how `getMudletInfo()` does the display.

Reply:
The previous one had to be reworked to handle the different way the data was changed and it had some short-comings - using the `Host::postMessage(...)` actually produced a poor layout for the information (the hanging indent meant it took up more space and the colouration didn't help and there wasn't sensible line breaking for what is a long line of text - as `postMessage(...)` expects multi-line texts to be pre-broken into lines).

Reply:
A different problem unrelated to adding CP437, I think. The old layout is far more compact than the new one, and more compact = good, less text to read.